docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -0,0 +1,315 @@
+interventions. The simulation tracks simulants through health states
+from age ~30 to 125 years (or death), capturing progression through
+preclinical AD, mild cognitive impairment (MCI) due to AD, and three
+stages of dementia due to Alzheimer's disease.
+
+2.1 Scenarios
+-------------
+
+1. **Reference Scenario:** Present-day conditions with minimal BBBM
+   uptake or disease-modifying therapies, including current
+   cerebrospinal fluid (CSF) and amyloid-positron emission tomography
+   (PET) diagnostic pathways
+2. **Alternative Scenario 1:** Introduction of BBBM testing for at-risk
+   preclinical populations (no intervention)
+3. **Alternative Scenario 2:** BBBM testing plus hypothetical
+   intervention that prevents, delays, or slows disease progression
+
+2.2 General Modeling Strategy
+------------------------------
+
+Based on literature and GBD, we conceive of Alzheimer's disease (AD) as
+comprising a six-stage progression:
+
+  **Susceptible → Preclinical AD → MCI due to AD → Mild AD → Moderate AD
+  → Severe AD**
+
+The last three stages correspond to the three sequelae (mild, moderate,
+severe) of the GBD cause "Alzheimer's disease and other dementias." We
+will have to separate AD out from other dementias in the GBD data, and
+we will need non-GBD data sources to inform our modeling of preclinical
+AD and MCI due to AD. Furthermore, reality may be a bit more complicated
+than the simple one-directional progression depicted above, but the
+assumption of no recovery from any state might be sufficient for our
+purposes.
+
+The basic plan for the design of the simulation is as follows:
+
+- Use forecasted population estimates
+
+  - We have data on 'population', 'deaths', 'migration', and 'births'
+    from FHS that can inform the age structure in the population out to
+    year 2100
+  - Based on GBD data, the incidence of AD within each age group is
+    pretty stable over time, so we are **not** planning on using
+    forecasted data for Alzheimer's disease
+
+- Only simulate people who will eventually get AD
+
+  - This drastically reduces population size and hence compute resources
+  - We will need to "work backwards" from GBD's Alzheimer's estimates
+    and the population forecasts to
+    determine how many people to add on each time step
+  - We will need to do some calculations outside the simulation to
+    account for false positive tests and people who don't progress from
+    preclinical AD or MCI to dementia due to AD
+
+- On top of the population model, we will add an Alzheimer's disease
+  progression model, a testing and diagnosis model, and a treatment
+  model, as detailed in the next section
+
+- We're doing a test run with mock-ups of all components and a full
+  population (not just simulants with AD) to get an idea of the runtime
+  for simulating 80 years in 10 locations
+
+3.0 Simulation Components
+++++++++++++++++++++++++++++++++++++
+
+.. list-table:: Simulation Components
```

Comment:
I mostly just copied this table from the AI -- I wasn't sure how useful the different columns are, so please let me know if you think it could use some changes!